docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,30 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking the DeepCell Kiosk
+---------------------------------------------------------
+
+The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To reproduced the cost and timing benchmarks reported in the DeepCell Kiosk paper, please refer to our `figure creation repository <https://github.com/vanvalenlab/kiosk-console/pull/github.com/vanvalenlab/publication-figures>`_. To run your own benchmarking, please read below.
+
+1. If you don't already have a cloud storage bucket devoted exclusively to DeepCell Kiosk benchmarking runs, you should create one now. It's fine to reuse this bucket for multiple benchmarking runs, but don't use it as a storage bucket for normal DeepCell Kiosk cluster operations.
+
+2. Checkout the ``benchmarks`` branch of the ``kiosk-console`` repository. This branch is almost identical to the version 1.2.0 release of the ``kiosk-console`` repo, except for minor configuration differences. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
```

Comment:
Reading it, the `benchmarks` branch reference doesn't seem useful here. Maybe we include that in the reproducibility section, and cut it from here?

```suggestion
2. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
```

Reply:
What do you mean by the reproducibility section exactly?